xml/bp_troubleshooting_tips.xml

```diff
@@ -781,4 +781,32 @@ keep_jobs: 1</screen>
    </procedure>
   </tip>
  </sect1>
+ <sect1 xml:id="smaster-new-node-failure">
```

Comment:
Since we are already documenting it above:

```
&prompt.smaster;zypper install ceph-salt
&prompt.smaster;systemctl restart salt-master.service
&prompt.smaster;salt '*' saltutil.sync_all
```
I vote to remove this from the troubleshooting tips.